core/modules/layout/layout.admin.inc

```diff
@@ -94,7 +94,34 @@ function layout_list_page() {
     array('data' => t('Operations'), 'class' => array('layout-operations')),
   );
 
+  $help_items = array();
+  $help_items[] = t('To determine which layout is to be used for a page, the path is compared to those shown here.');
```

Comment:
> To determine which layout is to be used for a page, the path is compared to those shown here

We should put the point before the reason. `The path for the current page will be compared to those listed here to determine which layout should be used."

Reply:
I think this was addressed in a recent PR, so marking this as resolved.